### PR TITLE
Feat: Initial Commit P0 Screens

### PR DESCRIPTION
**Pull request by jyothishp994**

### CODE REVIEW

**Comment by jyothishp994** [comment]
Please review and provide the suggestions. Below are the links
Home page- https://quizmine.netlify.app/
Categories- https://quizmine.netlify.app/categories.html
Question- https://quizmine.netlify.app/question.html
Result-https://quizmine.netlify.app/result.html
Rules-https://quizmine.netlify.app/rules.html

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+.answer-opt.btn.right-answer{
+    background-color: green;
+}
+.answer-opt.btn.wrong-answer{
+    background-color: red;
+}
+.quiz-card-question{
+    display: flex;
+    flex-direction: column;
+    background-color: #231F20;
```
Comment (spraveenofficial) [inline css/style.css:149]: You can make variables for colors.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    flex-direction: column;
+    gap:2rem;
+}
+
+
+.header-final-result{
+    align-self: center;
+    font-size: 3.5rem;
+    padding: 1rem;
+    font-family: 'Rowdies', cursive;
```
Comment (spraveenofficial) [inline css/style.css]: For font families too.

File: categories.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+</head>
+<body>
+    <header class="header">
+        <div class="headertitle">Quiz Mine</div>
+        <div class="header-sub-title">Categories</div>
+    </header>
+    <main class="quiz-card-main">
+       <div class="quiz-card">
+           <div class="quiz-card-wrapper">
+               <img class="quiz-card-img" src="./assets/politics.jpg" alt="image not available" />
```
Comment (Snigdha17) [inline categories.html:18]: You can give meaningful image descriptions in alt

File: categories.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
```
Comment (Snigdha17) [inline categories.html:7]: The assignment mentions that the component library css should be used when building the macro and mini projects. Are you planning to add it to your quiz project?

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+.header-sub-title{
+    padding: 1rem;
+    font-size: 2rem;
+    font-family: 'Rowdies', cursive;
+    align-self: center;
+}
+.quiz-card-main{
+    display: flex;
+    justify-content: center;
+    gap:1%;
```
Comment (sanket4120) [inline css/style.css]: use rem or pixel value in the gap. Due to percentage there isn't any row space between cards.

File: css/style.css (hunk 1 above)
Comment (jyothishp994) [reply css/style.css:149]: Updated the stylesheet to use var for colors.

File: css/style.css (hunk 2 above)
Comment (jyothishp994) [reply css/style.css]: Updated the stylesheet to use var for font families.

File: categories.html (hunk 3 above)
Comment (jyothishp994) [reply categories.html:18]: Updated meaningful descriptions in alt

File: categories.html (hunk 4 above)
Comment (jyothishp994) [reply categories.html:7]: Updated the code to make use of component from library

File: css/style.css (hunk 5 above)
Comment (jyothishp994) [reply css/style.css]: Correct the code to use rem, now this is working as expected

**Review by sanket4120 — APPROVED** [review]

**Review by Snigdha17 — APPROVED** [review]

**Pull request merged**